packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -178,19 +179,36 @@ export function registerSAPFonts() {
     IconPool.registerFont(suiteTheme);
 }
 
+/**
+ * Create Respource Bundle based on the scenario.
+ *
+ * @param scenario scenario to be used for the resource bundle.
+ */
+export async function loadI18nResourceBundle(scenario: string): Promise<ResourceBundle> {
+    if (scenario === 'ADAPTATION_PROJECT') {
+        const manifest = await getManifestAppdescr();
+        const enhanceWith = (manifest.content as { texts: { i18n: string } }[])
+            .filter((content) => !!content.texts?.i18n)
+            .map((content) => {
```

Comment:
You can return directly

```ts
 .map((content) => ({ bundleUrl: `../${content.texts.i18n}` }));
```

Reply:
Fixed with: e8c55b97e2800ab6e850e36c5b1a91f62a8be236